schemas/xml/AAS.xsd

```diff
@@ -703,16 +703,16 @@
         </xs:simpleType>
       </xs:element>
       <xs:element name="valueType" type="dataTypeDefXsd_t"/>
-      <xs:element name="value" type="xs:string" minOccurs="0" maxOccurs="1"/>
+      <xs:element name="value" type="xs:anySimpleType" minOccurs="0" maxOccurs="1"/>
```

Comment:
anySimpleType is the old version, in V3.0RC02 this was changed (see https://github.com/admin-shell-io/aas-specs/issues/210)  
Type of Propery/value is value „any xsd atomic type as specified via DataTypeDefXsd“ (https://industrialdigitaltwin.org/wp-content/uploads/2022/06/DetailsOfTheAssetAdministrationShell_Part1_V3.0RC02_Final1.pdf)

![grafik](https://user-images.githubusercontent.com/59824205/188120211-ea70faee-5ba7-4920-87b5-f7bbbe35e52c.png)